session/channel_test.go

```diff
@@ -192,6 +201,15 @@ func Test_SubUnsubChUpdate(t *testing.T) {
 	err = ch.UnsubChUpdates()
 	require.Error(t, err)
 
+	wantMessage = session.ErrNoActiveSub.Error()
+	assert.Equal(t, perun.ClientError, err.Category())
+	assert.Equal(t, perun.ErrV2ResourceNotFound, err.Code())
+	assert.Contains(t, err.Message(), wantMessage)
+	addInfo1, ok := err.AddInfo().(perun.ErrV2InfoResourceNotFound)
+	require.True(t, ok)
+	assert.Equal(t, "unsubscription to channel updates", addInfo1.Type)
```

Comment:
```suggestion
	assert.Equal(t, "subscription to channel updates", addInfo1.Type)
```

Reply:
Could you also add assertions on the returned error for the sub tests `Sub_channelClosed` and `Unsub_channelClosed`.